Source/DafnyLanguageServer/Program.cs

```diff
@@ -27,6 +27,8 @@ public class Program {
             // ReSharper disable once AccessToModifiedClosure
             .WithDafnyLanguageServer(configuration, () => shutdownServer!())
         );
+        // Prevent any other parts of the language server to actually write to standard output.
+        Console.SetOut(new LogWriter());
         shutdownServer = () => server.ForcefulShutdown();
         await server.WaitForExit;
```

Comment:
The `TextWriter` is disposable, so make sure all resources are freed.
```suggestion
        // Prevent any other parts of the language server to actually write to standard output.
        await using var logWriter = new LogWriter();
        Console.SetOut(logWriter);
        shutdownServer = () => server.ForcefulShutdown();
        await server.WaitForExit;
```